.gitignore

```diff
@@ -16,4 +16,6 @@ Publish
 /packages
 tools
 /node_modules
-/.output
\ No newline at end of file
+/.output
+
+.env
```

Comment:
Is this to avoid a virtual environment or [environment definition files](https://code.visualstudio.com/docs/python/environments#_environment-variable-definitions-file)?

Reply:
I created a virtual environment to keep development packages (Black, etc.) but didn't want to commit it to git.

Reply:
So the common/recommended practice is to name the environment `.venv` so as to not conflict with environment variable definition files. (I'm also noticing while I read the Functions docs that they unfortunately document `.env`.)

Reply:
Oh! I learned it from them.

Reply:
[pipenv uses .venv](https://pipenv.readthedocs.io/en/latest/advanced/#custom-virtual-environment-location) if you set `PIPENV_VENV_IN_PROJECT`.

Reply:
Virtual environments are no longer required for Python Functions development. However, we still recommend using them. There's a WI on me to update the guidance in the docs. Given that there's still a mix of folks using both `.env' and `.venv', my suggestion would be to include both here if there's no downside.

Reply:
Probably better to just include `.venv`. Avoids the potential of a name collision with `dev.env` and the like.